docs/dev-local-node.md

```diff
@@ -11,10 +11,7 @@ After pulling down all the code we'll spin up a docker container that runs a loc
 
     docker-compose -f support/test/integration/docker-compose.yml up --abort-on-container-exit --build --force-recreate geth-dev-node
 
-Verify the mapped ports.  
-We are interested in:  
-* HTTP: 47624  
-* WS: 47625 (WS isn't critical)
+Verify the mapped ports. We are interested in, HTTP: 47624 and WS: 47625 (WS isn't critical). Following is output via `docker ps`
```

Comment:
I see you didn’t like my formatting ;)

Reply:
Let’s leave the bulleted list in place — it’s a little easier to read.

Reply:
ok, I'll add back the formatting